### PR TITLE
created SETTING_UP_A_DEV_ENVIRONMENT.md

### PR DESCRIPTION
**Pull request by dangreenisrael**
Issue: N/A

## What I did

I created a walkthrough for setting up a dev environment with a demo project

## Why I did it

It took me a while to get my dev environment set up, so I thought I would codify the steps needed to do it.

### CODE REVIEW

File: SETTING_UP_A_DEV_ENVIRONMENT.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Please have the latest versions of the following on your machine
+
+-   node
+-   npm
+-   yarn
+-   create-react-app
+-   @storybook/cli
+
+Yes, we are using `npm` not `yarn` but you still need the latest version of both as there is a bootstrap script that uses both
```
Comment (Hypnosphi) [inline SETTING_UP_A_DEV_ENVIRONMENT.md]: @ndelangen should a separate version be written for `release/3.3` branch, which is yarn-only?

Comment (dangreenisrael) [reply SETTING_UP_A_DEV_ENVIRONMENT.md]: I could just update it to use yarn as opposed to npm. Would that be preferable? I wasn't sure what the project's stance was going to be on yarn with the new npm5 and lock files, etc.

Comment (dangreenisrael) [reply SETTING_UP_A_DEV_ENVIRONMENT.md]: I should have some time tomorrow

Comment (dangreenisrael) [reply SETTING_UP_A_DEV_ENVIRONMENT.md]: @Hypnosphi  Please see this PR: https://github.com/storybooks/storybook/pull/1864

It is the same walkthrough, updated to work with version 3.3.0-alpha.0 and yarn

**Comment by dangreenisrael** [comment]
I'm closing this, as I think it is superseded by https://github.com/storybooks/storybook/pull/1864

**Pull request closed without merging**